src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -929,25 +929,20 @@ private boolean areDatesOfPreviousCurrentAndNextMonthEnabled(WebElement dateBox)
      * Navigate the datepicker associated with {@code dateBox} to the specified {@code date}.
      *
      * @param dateBox is a {@link WebElement} that triggers a datepicker
-     * @param date is a {@link Calendar} that specifies the date that needs to be navigated to
+     * @param date is a {@link LocalDate} that specifies the date that needs to be navigated to
      * @return true if navigated to the {@code date} successfully, otherwise
      *         false
      * @throws ParseException if the string in {@code dateBox} cannot be parsed
      */
-    private boolean navigate(WebElement dateBox, Calendar date) throws ParseException {
+    private boolean navigate(WebElement dateBox, LocalDate date) throws ParseException {
 
         click(dateBox);
+        LocalDate selectedDate = LocalDate.parse(
+                dateBox.getAttribute("value"), DateTimeFormatter.ofPattern("EEE, dd MMM, yyyy"));
```

Comment:
Can we use `TimeHelper.parseLocalDateTimeForSessionsForm(dateBox.getAttribute("value"), 0, 0)` so as not to hardcode the pattern here?

Even better - extract a new method `TimeHelper#parseLocalDateForSessionsForm` and refactor `TimeHelper#parseLocalDateTimeForSessionsForm` to use the new method for the parsing of the date part and `LocalTime#of(int, int)` with `Integer#parseInt` for the parsing of the time part (then combine the two with `LocalDateTime#of(LocalDate, LocalTime)`). Then, use the new method `TimeHelper#parseLocalDateForSessionsForm` here.

Similar refactoring can be done for `TimeHelper#combineDateTime` - it can also use `TimeHelper#parseLocalDateForSessionsForm` for the parsing of the date part, custom logic for parsing of the time part, and `LocalDateTime#of(LocalDate, LocalTime)` to combine the two.

After making these changes, the constant `EEE, dd MMM, yyyy` will appear in only one place, instead of the current four different places (which are all related to the sessions form). Can also make a new issue for these changes if you don't feel like doing it in this PR.